Partner Manager 2.16.0 (#200)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoint-as2-send.adoc

|Protocol that the endpoint supports.
|x

|Endpoint name
|Automatically generated name used to identify the endpoint, which you can subsequently modify manually.

> |x

Reviewer: Would it be more clear to use "Yes" instead of "x" in the Required column?

|Description
|User-supplied value that describes the purpose of the endpoint.
|

|<Host> AS2 identifier
|Your organization’s AS2 identifier.
|x

|Partner AS2 identifier

> |Your partner’s AS2 identifier. Appears if the endpoint is used to send transactions directly to a partner without going through a third-party connection.

Reviewer: ```suggestion
|Your partner’s AS2 identifier, which appears if you use the endpoint to send transactions directly to a partner without a third-party connection.
```

|x

|<VAN> AS2 identifier

> |Your VAN’s AS2 identifier. Appears if the endpoint is used to send transactions to a third-party connection. 

Reviewer: ```suggestion
|Your VAN’s AS2 identifier, which appears if you use the endpoint to send transactions to a third-party connection. 
```

|x

|Partner AS2 URL
|Your partner’s or third-party connection's AS2 URL to which outbound EDI transmissions are sent.
|x

|Authentication type
a|Authentication type used to access the endpoint. Allowed values:

* None
+
Does not require backend applications to authenticate when connecting to this endpoint.
+
* Basic
+
Requires backend applications to use the username and password provided by your partner to connect to this endpoint.

> Enter this information in the Username and Password fields.

Reviewer: ```suggestion
Enter this information in the *Username* and *Password* fields.
```

For more information about host keystores, see xref:manage-as2-host-keystores.adoc[Managing AS2 Host Keystores].
|x

|Encryption algorithm
|xref:as2-endpoints-algorithms.adoc[Algorithm] to use when encrypting the message to send.
|x

|Signature algorithm

> |xref:as2-endpoints-algorithms.adoc[Algorithm] to use when setting the message integrity check value.

Reviewer: ```suggestion
|xref:as2-endpoints-algorithms.adoc[Algorithm] to use when setting the message-integrity check value.
```

|x

|Request receipt

> a|Whether you want the partner to return a Message Disposition Notification (MDN) receipt, which confirms the receipt of the message.

Reviewer: ```suggestion
a|Partner returns a Message Disposition Notification (MDN) receipt, which confirms receipt of the message.
```

Reviewer: Do you need the phrase after the comma?

|x

|MDN response
|The AS2 messages sent from Partner Manager always request the receiver to send a Synchronous MDN receipt back. The default is `Sync`.
|x

|MDN signature algorithm

> |Request for a partner to respond with a signed Message Disposition Notification (MDN) using this xref:as2-endpoints-algorithms.adoc[algorithm] for calculating MIC.

Reviewer: ```suggestion
|Partner responds with a signed Message Disposition Notification (MDN) using this xref:as2-endpoints-algorithms.adoc[algorithm] for calculating MIC.
```
